.github/workflows/whats-new-automation.yml

```
@@ -25,10 +26,14 @@ jobs:
         run: |
           echo "Reason: ${{ github.event.inputs.reason }}"
 
+      - name: Azure OpenID Connect
+        id: azure-oidc-auth
+        uses: ./.github/actions/azure-oidc-auth
+
       - uses: dotnet/docs-tools/WhatsNew.Cli@main
         env:
           GitHubKey: ${{ secrets.GITHUB_TOKEN }}
-          OspoKey: ${{ secrets.OSPO_KEY }}
+          OspoKey: ${{ steps.azure-oidc-auth.outputs.access-token }}
```

The environment variable here needs to change.  See https://github.com/dotnet/docs-tools/blob/main/WhatsNew.Infrastructure/Services/ConfigurationService.cs#L43

```suggestion
          AZURE_ACCESS_TOKEN: ${{ steps.azure-oidc-auth.outputs.access-token }}
```